package/kernel/r8125/patches/0001-ignore-rss-log.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/r8125_rss.c
++++ b/src/r8125_rss.c
+@@ -91,7 +91,7 @@ int rtl8125_get_rxnfc(struct net_device
+         struct rtl8125_private *tp = netdev_priv(dev);
+         int ret = -EOPNOTSUPP;
+ 
+-        netif_info(tp, drv, tp->dev, "rss get rxnfc\n");
++        netif_dbg(tp, drv, tp->dev, "rss get rxnfc\n");
```

Comment:
Should we silence this on [rtl8126](https://github.com/openwrt/rtl8126/blob/7262bb22bd3a20dfb47124c76d6b11587b3c5e78/r8126_rss.c#L94) too?

Reply:
I don't have the hardware so I don't know.

Reply:
> I don't have the hardware so I don't know.

I do but I couldn't get that logged at all.
Do I have to do something or configure something specifically to get it?

Reply:
> Do I have to do something or configure something specifically to get it?

It's a little strange, but the same issue can be found through a search:
https://github.com/search?q=rss+get+rxnfc&type=issues